compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/window/WindowState.desktop.kt

```diff
@@ -227,20 +229,25 @@ private class WindowStateImpl(
                     it.position.isSpecified,
                     it.position.x.value,
                     it.position.y.value,
-                    it.size.width.value,
-                    it.size.height.value,
+                    it.size.isSpecified,
```

Comment:
We should store `width.isSpecified` and `height.isSpecified` separately. We can create `DpSize(Dp.Unspecified, 40.dp)`

Reply:
There's no special handling needed for `Dp` because `Dp(dp.value)` works correctly for `Dp.Unspecified` too. The problem with `DpSize` is that `DpSize.width` and `DpSize.height` throw on `DpSize.Unspecified`.

Reply:
Ok